test/bpf/Makefile

```diff
@@ -19,8 +19,10 @@ CLANG ?= $(QUIET) clang
 LLC ?= llc
 
 BPF_TARGETS := elf-demo.o
-TARGETS := $(BPF_TARGETS) unit-test
-all: $(TARGETS)
+ALL_TESTS := unit-test nat-test
+TARGETS := $(BPF_TARGETS) $(ALL_TESTS)
+
+all: $(TARGETS) unit-tests
```

Comment:
@kkourt this line + the unit-tests dependency on `ALL_TESTS` below -> depending on `unit-test` should allow us to remove the direct `unit-test` call from the top-level makefile.